function.js

```diff
@@ -0,0 +1,105 @@
+// 1 Create a function that takes 3 arguments 
+function threeArg(a,b,c){
+    let d = a + b + c ;
+    return d;
+}
+
+var sum = threeArg(3,4,3)
+console.log(sum);
+
+// 2  function  colorCar
+function colorCar(letter ,color, auto){
+    let syn = letter +" " + color + " " + auto ;
+    return syn;
+}
+
+var clr = colorCar( "a","red","car");
+console.log(clr);
+
+// 3  a function that takes the object as a paramete
+
+function obj(objec){
+return objec;
+}
+
+let obje = {
+    name : "Abdul",
+    old : 30
+}
+
+let variable = Object.keys(obje);
+let value = Object.values(obje);
+
+for(let i = 0; i < variable.length; i++){
+    console.log(obj(variable[i]));
+    console.log(obj(value[i]));
+}
+
+//  4 receives a color
+function vehicleType(color, code){
+    if (code == 1){
+        console.log("a " + color + " car");
+    }else if (code == 2){
+        console.log("a " + color + " motobike")
+    }
+}
+
+vehicleType("blue", 2)
+
+// 5 without the if
+3 === 3 ? console.log("yes") : console.log("no");
+
+// 6 a function called vehicle with parameter called age
+function vehicle(color , code , age){
+    if(code == 1)
+    console.log("a "+ color + " car");
+
```

Comment:
Ah, I think you forgot the `}`

I would also recommend to always add brackets in your `if-statement` (even if it is only one line). It helps avoid silly mistakes:

```
if(code == 1) {
    console.log("a "+ color + " car");
}
```